modules/minileven/theme/pub/minileven/content-gallery.php

```diff
@@ -75,4 +80,4 @@
 	</footer><!-- #entry-meta -->
 </article><!-- #post-<?php the_ID(); ?> -->
 
-<?php comments_template( '', true ); ?>
```

Comment:
I don't think we have a strong preference to newlines, so let's keep those as they were to remove unnecessary changes from the diff.